neurolang/probabilistic/tests/test_marg_query_resolution.py

```diff
@@ -4,7 +4,11 @@
 from ...logic import Conjunction, Implication
 from ...relational_algebra import ColumnStr, NamedRelationalAlgebraFrozenSet
 from ...relational_algebra_provenance import ProvenanceAlgebraSet
-from .. import dichotomy_theorem_based_solver, weighted_model_counting
+from .. import (
+    dalvi_suciu_lift,
+    dichotomy_theorem_based_solver,
```

Comment:
maybe this should be renamed to `small_dichotomy_theorem_based_solver`

Reply:
done